src/components/Badge/Badge.meta.tsx

```diff
@@ -0,0 +1,12 @@
+import {Badge, BADGE_PRIORITY} from '.';
+import Registry from '@ui-autotools/registry';
+
+const BadgeMetadata = Registry.getComponentMetadata(Badge);
+BadgeMetadata.nonReactStrictModeCompliant = true;
+
+BadgeMetadata.addSim({
+  title: 'render',
+  props: {
+    priority: BADGE_PRIORITY.default,
```

Comment:
We should probably also add a `children` prop with some text

Reply:
But `children` do not exist in BadgeProps

Reply:
🤔 but we use `children` in `Badge.st.css`